plotly/Make_a_FigureFactory.md

```diff
@@ -0,0 +1,113 @@
+# Add Your Figure Factory to the Plotly [Python Library](https://plot.ly/python/)
+
+If you have ever wanted to contribute to the Plotly Python Library by adding a new chart type we don't have, now you have the resources to do so. This README will help you get started cloning the plotly.py repo, forking a new branch, creating a new figure factory, and pushing your results to the cloud to get feedback for merging. Just follow all these steps and you'll be ready to go.
+
+## Getting Started:
+1. Clone the `plotly.py` repo locally and then check out the master branch.
+
+```
+$ git clone git@github.com:plotly/plotly.py.git
+$ git fetch origin
+$ git checkout master
+```
+
+2. Checkout a new branch and give it an appropriate name.
+
+```
+$ git checkout -b "my-new-ff"
+```
+
+## Create A figure_factory file
+1. Creating python file and updating `__init__.py`
```

Comment:
we should keep all the directions in the same tense so
## Create A figure_factory file
  1. Create a python file and update `__init__.py`

Reply:
you could probably split this step into 2 steps:
creating a python file and updating __init__.py